gulpfile.js

```diff
@@ -130,6 +133,21 @@ gulp.task( 'bundle:generate',
 
 // Documentation. -------------------------------------------------------------
 
-const docsBuilder = ckeditor5DevCompiler.docs( config );
+// Documentation. -------------------------------------------------------------
+
+const ckeditor5DevDocs = require( '@ckeditor/ckeditor5-dev-docs' );
+const docsBuilder = ckeditor5DevDocs.docs( config );
+
+gulp.task( 'docs', [ 'compile:js:esnext' ], ( done ) => {
```

Comment:
To add to this – most likely, we'll need a function which is able to take functions which generates streams (sub tasks) and make one big stream out of that. Gulp does this internally and `runSequence` uses that, but we need to bypass gulp. Otherwise, we'll get crazy.